sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -50,7 +50,7 @@
 def test_split_code_and_text_blocks():
     """Test if a known example file gets properly split"""
 
-    blocks = sg.split_code_and_text_blocks('examples/just_code.py')
+    blocks,_ = sg.split_code_and_text_blocks('examples/just_code.py')
```

Comment:
I have pep8 checks in my editor, but I find this one annoying so it's disabled. I'll fix to be consistent with this repository.